src/solvers/mg_solver.hpp

```diff
@@ -386,20 +394,20 @@ class MGSolver {
       PARTHENON_FAIL("Unknown solver type.");
     }
 
-    auto decorate_task_name = [partition, level](const std::string &in, auto b) {
-      return std::make_tuple(in + "(p:" + std::to_string(partition) +
-                                 ", l:" + std::to_string(level) + ")",
-                             1, b);
-    };
+//    auto decorate_task_name = [partition, level](const std::string &in, auto b) {
+//      return std::make_tuple(in + "(p:" + std::to_string(partition) +
+//                                 ", l:" + std::to_string(level) + ")",
+//                             1, b);
+//    };
```

Comment:
Any particular reason to keep this commented (rather than reintroducing or removing)?